wdl-grammar/src/experimental/tree.rs

```diff
@@ -203,6 +205,10 @@ pub enum SyntaxKind {
     InputSectionNode,
     /// Represents an output section node.
     OutputSectionNode,
+    /// Represents a braced command section node.
+    BracedCommandSectionNode,
+    /// Represents a heredoc command section node.
+    HeredocCommandSectionNode,
```

Comment:
Just a thought: there's a good deal of redundant code created by treating these as separate Nodes. Does it make sense to consolidate the code for these cases into just a plain `CommandSectionNode`? That may prove more complicated than it's worth. I'm not sure.

Reply:
> don't you need to differentiate because it changes how contained items are interpreted?

At this point we've already parsed the interpolation of the command contents, so it shouldn't matter for the CST.